### PR TITLE
Fix: Remove search box outline

### PR DESCRIPTION
**Pull request by Onokaev**
## Overview
Solves #1427 

### Demo
image

image

image

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

### CODE REVIEW

File: src/app/views/sidebar/history/History.tsx [hunk 1]
```diff
@@ -101,9 +102,13 @@ export class History extends Component<IHistoryProps, any> {
...
+  }
+
+  private sortItems = ( items: any ) => {
     items
       .sort(dynamicSort('createdAt', SortOrder.DESC))
-      .forEach((value, index) => {
+      .forEach((value : any, index: any) => {
         value.index = index;
       });
     return items;
```
Comment (thewahome) [inline src/app/views/sidebar/history/History.tsx:114]: Let's use the types here instead of any especially since we know what we are sorting

Comment (Onokaev) [reply src/app/views/sidebar/history/History.tsx:114]: Making the changes

File: src/app/views/app-sections/ResponseMessages.tsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ export function responseMessages(graphResponse: IGraphResponse, sampleQuery: IQu
   if (odataLink) {
     return (
       <MessageBar messageBarType={MessageBarType.info}>
-        <FormattedMessage id={'This response contains an @odata property'} />: @odata.{odataLink!.name}
+        <FormattedMessage id={'This response contains an @odata property.'} />: @odata.{odataLink!.name}
```
Comment (ElinorW) [inline src/app/views/app-sections/ResponseMessages.tsx:52]: Why was the full stop added? I don't think we should have it before a colon

Comment (thewahome) [reply src/app/views/app-sections/ResponseMessages.tsx:52]: Looks like it is how it went into the all of the localization files

**Review by ElinorW — APPROVED** [review]

**Pull request merged**